### DISCUSSION TITLE
Is there any way to expedite or rescind my previous application for the education pack?

### DISCUSSION THREAD

**Discussion by JohnnyShi199** [community/community#44136] [GitHub Education] [answered]
When I submitted the application, I thought I maybe enclose a wrong photo and want to resubmit again. Then I met 502 error and cannot resubmit the correct material again. The post after I submitted said that I would be contacted in a day but after one day I didn't receive any reply...
I am really in a hurry to get GitHub education pack because I need it to finish my assignment before ddl! I cannot wait for two review cycles. So is there any way to expedite or rescind my previous application for the education pack? Thank you for your kind reply!

**Comment by melvincwng** [comment]
Hey @NeedyChild 👋🏻, I understand your concerns with regards to your school assignment.

Unfortunately, as mentioned by one of the GitHub staff, from discussion #17814:

> ### How long after I submit my application will it be approved?
> Applications are generally processed within a few days but may take longer in peak periods such as the start of a new semester. Please note our support team is unable to expedite the review process, so opening a support ticket will not decrease the review time.
> At the busiest times of the year, application review times can be up to six weeks

In summary, **there doesn't seem to be a way to expedite the review process**. My recommendation here is for you to wait out the approval period, and once your application gets rejected, quickly apply for the next cycle for review. 

You might also want to check with your school or teachers if any extension of the homework/assignment is possible.

Hope this answers your question, take care 😃

**Reply by melvincwng** [reply]
@NeedyChild, with regards to rescinding your application, you can consider contacting [GitHub Education Support](https://support.github.com/contact/education). But they usually take between a few days to weeks to reply to a ticket.

So I'm not exactly sure if it would be a right fit for your particular scenario.

All the best.

**Reply by shishir-katakam** [reply]
I thought I've to accept my education pack activation after applying but then I found that there's no activation required.Im in my B.Tech 1st year and I don't actually need education pack now,so can I cancel it and really again in the  2nd year?

**Comment by samus-aran** [comment, marked as answer]
Hi @NeedyChild 👋 

As @melvincwng has commented, we are not able to review and expedite individuals' applicants. 

I would like to ask you to visit [GitHub Student Developer Pack](https://education.github.com/benefits) and see if you have been successful. You may not have received your application result email due to spam filters or your academic mail settings which can block external mail.

Let us know how you get on 👍

**Reply by JohnnyShi199** [reply]
Thanks a lot for the reply from both of you! I finally solve this by switching from chrome browser to safari browser.

By using Google chrome I can only see the 502 error. But in safari browser, I see the material I submitted before is correct and my request is approved!! I hope this message can help others who meet the same problem.

**Reply by samus-aran** [reply]
Awesome to hear this (excluding the 502 error 😄 ) 🎉

**Comment by rayennecandido** [comment]
The 502 Bad Gateway is an error that indicates problems accessing a website. Generally, the code arises when there is a failure in communication between two or more servers that relay data to each other to load the page. Also, 504 Gateway Time-out occurs when the server, while acting as a gateway or proxy, failed to respond in time. The causes are linked, among other factors, to a possible overload of hits on the accessed site.
That is, the 502 Bad Gateway error does not refer to problems with the user's notebook or PC. Despite this, it is possible to try some procedures to make sure that, in fact, it is not a false alarm. See, in the following tutorial, what a 502 Bad Gateway is, in addition to checking out six ways to resolve the error and be able to access a website.
1. Refresh the page
Even if there is a glitch on the site, it could be temporary. The 502 Bad Gateway Nginx error can be caused when one server receives a bad response from another, but the next request may work. In this case, simply refreshing the site may do the trick. It is worth clicking the button directly in the browser or pressing the F5 key several times to check if the problem has been resolved.
2. Confirm that there is a problem on the site
If the problem is not resolved, it is possible to check if the fault also occurs with other people. Monitors like IsItDownRightNow (isitdownrightnow.com) can tell you if there's a sizable volume of complaints about the same site elsewhere. In general, if there is a problem, the tool will show an orange or red signal that confirms that there is a fault.

3. Switch browsers and clear cache
If the site is apparently working for other people, you need to look for solutions on the computer. Start by clearing your browser's cache and cookies. In Chrome, just go to the "History" menu in the settings and start a cleaning by checking only the cookies and cache options. Then try to access the site again. If it still doesn't come back, try opening it in another browser, such as Edge, Firefox or Safari on macOS.

4. Reboot the router
If the page insists on displaying the same error in multiple browsers with clean cookies and cache, you need to check the router. Restart the device to renew the connection and eliminate any network problems that could prevent the website from loading. Some devices have an on-off switch. If not, simply unplug the equipment and plug it back in a few seconds later.

5. Change the DNS
Another possible solution involves changing the DNS. This service is responsible for translating the website address to the IP of the corresponding server. If there is a failure in the process, it is possible that the user lands on a wrong server that is experiencing a communication failure. In general, the connection is configured with the DNS used by the internet provider, but it is possible to change it in the settings of the network adapter. Google's DNS (8.8.8.8 and 8.8.4.4), Cloudfare's (1.1.1.1 and 1.0.0.1) are two options known for their efficiency.

6. Update and restart your computer
If nothing works, it is possible that there is some conflict in the PC itself. Network faults can be corrected by a simple computer reset. Also, you can try to download pending Windows 10 updates. If there are many packages not installed, one of them may resolve an issue that is known and fixed but still needs to be applied.
